ztp/gitops-subscriptions/argocd/example/policygentemplates/group-du-standard-validator-ranGen.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  # The name will be used to generate the placementBinding and placementRule names as {name}-placementBinding and {name}-placementRule
+  name: "group-du-worker-validator"
+  namespace: "ztp-group"
+spec:
+  bindingExcludedRules:
```

Comment:
Without binding rules to match some type of common or group label this policy will apply to anything without ztp-done. Should it also include a binding rule for ztp-running so that it doesn't apply to (and fail against) the "local-cluster" and/or any other clusters the user added to the hub outside the scope of ZTP.

Reply:
If adding bindingRule for ztp-running, validator policy will be applied to cluster with ztp-running label, but this label is added when start remediating auto-generated UOCR, so at the time of creating UOCR, validator policy will be excluded in the list of managedPolicies so we cannot add the ztp-running bindingRules.
But yes, with only bindingExcludedRules, it will apply to the local-cluster and other clusters added outside of ZTP, so we want another bindingRules to exclude them.

It occurs to me that we actually need have 3 validator PGTs for each type of cluster, i.e.
    ztp-group-sno bindingRules for sno
    ztp-group-3node bindingRules for 3node
    ztp-group-standard bindingRules for standard
, then corresponding validator policy will be applied to the right type of cluster, otherwise, for example, group-du-worker-validator will be applied to the sno as well. It's a bit duplicate for sno and 3node validator policy as the only difference is different label...

Reply:
I think I understand and agree that we do need one per cluster type. With the use case where the user deploys both SNO and standard clusters, the hub will have both the SNO and standard DU validator created and right now they will both bind to newly deploying SNO and standard clusters. That could mean that the cluster never finished ZTP because only one of the two DU validators will ever be compliant. By adding a bindingRule for the matching group type (eg group-du-sno and group-du-standard we avoid this.